crates/aptos/src/account/create.rs

```diff
@@ -6,17 +6,20 @@
 //! TODO: Examples
 //!
 
-use crate::common::types::{
-    account_address_from_public_key, CliError, CliTypedResult, EncodingOptions, ExtractPublicKey,
-    PublicKeyInputOptions, WriteTransactionOptions,
+use crate::common::{
+    init::DEFAULT_FAUCET_URL,
+    types::{
+        account_address_from_public_key, CliConfig, CliError, CliTypedResult, EncodingOptions,
+        ExtractPublicKey, PublicKeyInputOptions, WriteTransactionOptions,
+    },
 };
 use aptos_crypto::{ed25519::Ed25519PrivateKey, PrivateKey};
 use aptos_rest_client::{Client as RestClient, Response, Transaction};
 use aptos_sdk::{transaction_builder::TransactionFactory, types::LocalAccount};
 use aptos_transaction_builder::aptos_stdlib;
 use aptos_types::account_address::AccountAddress;
 use clap::Parser;
-use reqwest;
+use reqwest::{self, Url};
```

Comment:
you import `Url` here, but elsewhere use `reqwest::Url` too

Reply:
I'll fix this